lib/create.js

```diff
@@ -0,0 +1,38 @@
+import {createHasher, hashCanonizedProof, stringToUtf8Bytes}
+  from '@digitalbazaar/di-sd-primitives';
+import {canonize} from './canonize.js';
+import {concat} from './helper.js';
+import {createVerifier} from './createVerifier.js';
+import {name} from './name.js';
+import {requiredAlgorithm} from './requiredAlgorithm.js';
+
+export function createCryptosuite({extraInformation = ''} = {}) {
```

Comment:
```suggestion
export function createCryptosuite({extraInformation = new Uint8Array()} = {}) {
```

See comment below as well.